extensions/amp-story/1.0/story-analytics.js

```diff
@@ -78,6 +93,9 @@ export class StoryAnalyticsService {
     /** @const @private {!./variable-service.AmpStoryVariableService} */
     this.variableService_ = getVariableService(win);
 
+    /** @private {EventsPerPageDef} */
+    this.pageEventCount_ = map();
```

Comment:
Can we improve the naming here?
`pageEventCount` of type `EventsPerPageDef` sounds weird: one is a count (so `number`?), and one has a single `event` when the other has multiple `events`